schema.graphql

```diff
@@ -1,124 +1,127 @@
-type AssetSourceUpdated @entity {
-  id: ID! 
-  asset: Bytes! 
+type Assetsourceupdated @entity {
+  id: ID!
+  asset: Bytes!
   source: Bytes!
   evtBlockTime: BigInt!
   evtBlockNum: BigInt!
   contractAddress: Bytes!
 }
 
-type BaseCurrencySet @entity {
-  id: ID! 
-  baseCurrency: Bytes!
-  baseCurrencyUnit: BigInt!
-  evtBlockTime: BigInt!
-  evtBlockNum: BigInt!
-  contractAddress: Bytes!
-}
+# type BaseCurrencySet @entity {
```

Comment:
These will need to be made lower case too, ie ~~BaseCurrencySet~~ = Basecurrencyset